### DISCUSSION TITLE
Unable to connect

### DISCUSSION THREAD

**Discussion by nunyakuforkthis** [chromiecraft/chromiecraft#7373] [General]
Okay, so I'm running Windows 8.1, with no hope of upgrading. HP AiO 23, with upgraded RAM. I see no reason why it shouldn't work, but retail just refuses to install on my system and running Windows 8, no official channels are willing to help me. PLEASE help, I MUST play again! title says it all. registered my account last night. I just finished torrenting ChromieCraft, the one with the fixed realmlist, everything runs fine. but when I put in my login info it just tries for about 30 seconds then returns the error "Unable to connect. Please try again later. If the problem persists, please contact technical support at: blizz info". pretty sure I'm supposed to use just my username, but I've tried both with and without my email address, just to be sure I wasn't doing it wrong.

**Comment by Helias** [comment]
Hi, you can contact me on discord helias#1661 and/or better, open a ticket https://www.chromiecraft.com/en/need-help/

are you able to login here https://chromiecraft.com/wp-login.php ?
did you try to download again the client from another source like from the direct link?

**Reply by nunyakuforkthis** [reply]
I am able to login to that page. unfortunately discord is a no go for me, having "suspicious activity" issues because I never use it.

**Reply by Helias** [reply]
Are you sure that the realmlist.wtf file is correct? did you copy/paste the entire file?
to be sure you could download this file https://shinworld.altervista.org/downloads/realmlist.wtf (to save it, just visit the page and type Ctrl+S)
and put it inside the directory WoW/Data/enUS/

**Reply by nunyakuforkthis** [reply]
I would prefer we continue here, honestly. I don't have space on my phone for more apps. I checked the realmlist, and yes it is as it should be. the version I torrented said it had the realmlist already set, but I checked it just the same when the login failed. 

in the efforts of being thorough, I downloaded the file and replaced the existing, attempted a login again. still negative.

**Reply by Helias** [reply]
check in someway if in your network is blocked the port 40000

**Reply by Helias** [reply]
meanwhile I recommend to try again downloading the client from the direct link, but not sure if it can solve

**Reply by nunyakuforkthis** [reply]
oookay, wilco. might be a little while, I'm a bit rusty on checking that stuff. could this maybe be a NAT type issue?

**Reply by nunyakuforkthis** [reply]
is the torrent less reliable than direct? it kept failing yesterday, which is why I switched to torrent.

**Reply by Helias** [reply]
`oookay, wilco. might be a little while, I'm a bit rusty on checking that stuff. could this maybe be a NAT type issue?`
Maybe, not sure
`is the torrent less reliable than direct?`
No, it should be ok, but sometimes redownloading the client (even still from torrent) could be a good boule check, changing the download source too

**Reply by Helias** [reply]
I think it's more a network issue than a client one

**Reply by nunyakuforkthis** [reply]
> I think it's more a network issue than a client one

that would not surprise me in the slightest, and unfortunately is a whole can of wyrms all by itself. comcast's methods of changing or even accessing info like that is becoming increasingly difficult on my outdated devices. and it would seem the workaround I found to look at and change port info has been patched out.

**Reply by nunyakuforkthis** [reply]
> check in someway if in your network is blocked the port 40000

okay, I'm fairly certain port 40000 is closed. I guess I will report back if/when I figure out how to open that up.

**Reply by Helias** [reply]
you could also check if with another network, like with hotspot and/or with another machine it works, to understand if the issue is the network or not

**Reply by nunyakuforkthis** [reply]
> you could also check if with another network, like with hotspot and/or with another machine it works, to understand if the issue is the network or not

I can try from a linux machine, but it would still be on the same network. thank you for your ideas and effort.

**Reply by Helias** [reply]
this could definitely help, let's try it

**Reply by Helias** [reply]
did you solve?

**Reply by nunyakuforkthis** [reply]
apologies for not getting back to you sooner, TLDR kids and updates, been a while since I last booted the linux. I just finished testing it, and negative. getting the same error. I assume this more or less confirms the port issue? is there some brute force method I can use to open that or do I have to go through the router controls?

**Reply by Helias** [reply]
are you running linux in a virtual machine still under windows? if so it will use the same network setting of your windows machine, if you are using it in dual boot it's a good test

but, I would ask you to verify if with another connection (ex. hotspot) you are experiencing the same

**Reply by nunyakuforkthis** [reply]
no, it is a separate machine, albeit very similar. windows 8.1 = HP AiO 23, ubuntu 23 = HP AiO 19. 

unfortunately the soonest I will have another network available to me will be this coming Sunday, whereupon we will be visiting with my parents, so I'm not sure how much time I'll be able to dedicate to troubleshooting while actually there.

**Reply by nunyakuforkthis** [reply]
side note: I was blown away by how easily the login client ran with wine on other machine. assuming that machine is experiencing the same issue, I may end up playing on that instead. I loathe using windows but thusly it has been the only feasible gaming platform for me. I don't want to complicate things though...I suppose the linux could be experiencing a different issue, but I'm not sure how to tell at this point. probably best to just continue troubleshooting what we started on, no?

**Reply by Helias** [reply]
I always used Linux for WoW and I did not have issues, so I hope that you will not have other issues
but so, do you get the same "Unable to connect" on Linux or are you encountering other errors? if it's the same, probably it's a network thing, but we'll know it on Sunday then

**Reply by nunyakuforkthis** [reply]
I get the same "Unable to connect" on Linux. I suppose I'll get back to you sometime Sunday evening. thank you again, for your continued efforts.
